Android.mk

```diff
@@ -24,14 +23,13 @@ rwildcard=$(wildcard $1$2) $(foreach d,$(wildcard $1*),$(call rwildcard,$d/,$2))
 
 include $(CLEAR_VARS)
 LOCAL_LDLIBS     := -llog
-LOCAL_CFLAGS     := -D"MOD_ID=\"{{ mod.id }}\"" -D"VERSION=\"0.1.0\"" -I"{{ mod.libil2cpp }}"
+LOCAL_CFLAGS     := -D'MOD_ID="{{ mod.id }}"' -D'VERSION="0.1.0"' -I'{{ mod.libil2cpp }}'
```

Comment:
I have only had problems with this approach when weird characters are in the mod.libil2cpp, please confirm that this is a valid replacement (TODO for me as well).